source/base/function_parser.cc

```diff
@@ -176,6 +179,22 @@ namespace internal
   {
     return erfc(value);
   }
+
+  // returns a random value in the range [0,1]
+  double mu_rand(double seed)
+  {
+    static bool uninitialized = true;
+    if (uninitialized)
+      {
+        if (seed > 0)
+          srand((unsigned int)seed);
```

Comment:
This makes no sense. If someone provides a positive seed, then you call `srand` every single time with the same argument, and you will also get the very same (not so random) number in the following call to `rand()`.